jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/QuarkusModeTest.java

```diff
@@ -120,51 +123,26 @@ public void from_withNoSettingsAndRunnerJarAndLib_shouldReturnLegacyJar() throws
     assertThat(result).isEqualTo(QuarkusMode.LEGACY_JAR);
   }
 
-  @Test
-  public void from_withNativePackaging_shouldReturnNative() {
-    // Given
-    projectProperties.put("quarkus.package.type", "native");
-    // When
-    final QuarkusMode result = QuarkusMode.from(project);
-    // Then
-    assertThat(result)
-        .isEqualTo(QuarkusMode.NATIVE)
-        .hasFieldOrPropertyWithValue("fatJar", false);
-  }
-
-  @Test
-  public void from_withLegacyJarPackaging_shouldReturnLegacyJar() {
-    // Given
-    projectProperties.put("quarkus.package.type", "legacy-jar");
-    // When
-    final QuarkusMode result = QuarkusMode.from(project);
-    // Then
-    assertThat(result)
-        .isEqualTo(QuarkusMode.LEGACY_JAR)
-        .hasFieldOrPropertyWithValue("fatJar", false);
-  }
-
-  @Test
-  public void from_withFastJarPackaging_shouldReturnFastJar() {
+  @DisplayName("Quarkus Packaging tests with different modes")
+  @ParameterizedTest(name = "{0}")
```

Comment:
You should adapt the test so that the test name can be defined as
`QuarkusMode from project with '{0}' packaging should return '{1}'`.

For example: `QuarkusMode from project with 'native' packaging should return 'NATIVE'`

Then the assertion should include `isFatJar`

So the method signature should look sth like 
```
void fromWithPackageType(String packageType, QuarkusMode expectedMode, boolean isFatJar)
```

You should also notice how in your initial implementation, both `String property`, and the `property` arguments make no sense in the scope of the test. If those were to be different (which aren't), then different tests should be expected.